apps/wallet/src/ui/app/shared/ModalDialog.tsx

```diff
@@ -2,23 +2,39 @@
 // SPDX-License-Identifier: Apache-2.0
 
 import { Dialog, Transition } from '@headlessui/react';
+import { cva, type VariantProps } from 'class-variance-authority';
 import { Fragment, type ReactNode, useEffect } from 'react';
 
 import { Heading } from './heading';
 
-export type ModalDialogProps = {
+const styles = cva(
+    [
+        'shadow-wallet-modal bg-white py-6 rounded-xl max-w-[85vw] max-h-[60vh]',
```

Comment:
do we have a need for this extra-wide modal? seems like we could just default to the small width for now

Reply:
no not necessary just thought it might be useful if we introduce other modals - I can remove it if thats better

Reply:
yea, I think we should just remove until we have a need  for it

Reply:
done!